cuml/linear_regression_demo.ipynb

```diff
@@ -86,17 +78,30 @@
     "### GPU"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cudf\n",
+    "from cuml import make_regression\n",
+    "from cuml import train_test_split"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "%%time\n",
-    "X_cudf = gd.DataFrame.from_pandas(X_train)\n",
-    "X_cudf_test = gd.DataFrame.from_pandas(X_test)\n",
+    "X, y = make_regression(n_samples=n_samples, n_features=n_features, random_state=0)\n",
```

Comment:
We dont need to create the regression dataset twice, we can use cudf function `from_pandas` to convert pandas dataframe to cudf dataframe

Reply:
My first approach was to compare both ways to generate data, via SKL and via cuML.

I have changed the code to create the date via cuML, and then copy the data to host using cuDF's to_pandas() method.

Fixed in local.